Sources/Build/BuildOperation.swift

```diff
@@ -334,16 +334,30 @@ public final class BuildOperation: PackageStructureDelegate, SPMBuildCore.BuildS
                 self.buildSystemDelegate?.preparationStepStarted(preparationStepName)
             }
             func didCompilePlugin(result: PluginCompilationResult) {
-                if !result.description.isEmpty {
-                    self.buildSystemDelegate?.preparationStepHadOutput(preparationStepName, output: result.description)
+                self.buildSystemDelegate?.preparationStepHadOutput(
+                    preparationStepName,
+                    output: result.commandLine.joined(separator: " "),
```

Comment:
Is this what the command lines for `swift build` also show, or do they go through some code that quotes whitespace and escapes special characters, etc?  If the same, then this seems fine, otherwise this code should do the same.

Reply:
affect the build system has a very different setup with BuildSystemCommand that parse llbuild commands. so not fully applicable here. do you have some specific utility in mind that can help?